zio/src/test/scala/org/scanamo/ScanamoZioSpec.scala

```diff
@@ -7,14 +7,15 @@ import org.scanamo.fixtures.*
 import org.scanamo.generic.auto.*
 import org.scanamo.ops.ScanamoOps
 import org.scanamo.query.*
-import org.scanamo.syntax.*
 import software.amazon.awssdk.services.dynamodb.model.DynamoDbException
 import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType.*
 import zio.Runtime.default.*
 import zio.stream.interop.catz.*
 import zio.stream.{ Sink, Stream }
 
 class ScanamoZioSpec extends AnyFunSpec with Matchers {
+  import org.scanamo.syntax.* // here for Scala 3, otherwise we get `===` from org.scalactic.TripleEqualsSupport
```

Comment:
Likewise, we've recently seen this problem before:

* https://github.com/scanamo/scanamo/pull/1805

As [mentioned there](https://github.com/scanamo/scanamo/pull/1805/files#r1731491275), this fixes a [compilation error](https://github.com/scanamo/scanamo/actions/runs/10582912126/job/29323695241#step:7:1420) relating to an apparent change in Scala 3's implicit precedence, that led to it picking the wrong `===` method:

```
[error] -- [E007] Type Mismatch Error: /home/runner/work/scanamo/scanamo/zio/src/test/scala/org/scanamo/ScanamoZioSpec.scala:27:25 
[error] 27 |        f <- farmers.get("name" === "McDonald")
[error]    |                         ^^^^^^^^^^^^^^^^^^^^^
[error]    |                         Found:    Boolean
[error]    |                         Required: org.scanamo.query.UniqueKey[?]
```